.github/workflows/ci.yml

```diff
@@ -871,8 +871,6 @@ jobs:
             - suite: suite-delta-lake-databricks133
               ignore exclusion if: >-
                 ${{ env.CI_SKIP_SECRETS_PRESENCE_CHECKS != '' || secrets.DATABRICKS_TOKEN != '' }}
-            - suite: suite-databricks-unity-http-hms
-              config: hdp3
```

Comment:
this one is unconditional, the other is conditional

does the unconditional pass? if yes, why not keep this one?